client/src/components/AdvancedSearch.js

```diff
@@ -177,7 +177,7 @@ class AdvancedSearch extends Component {
 
 		if (filter.key === "Organization") {
 			this.setOrganizationFilter(null)
-		} else if (filter.key === "Position type") {
+		} else if (filter.key === "Position Type") {
```

Comment:
Can we have some fancy way to compare to the "Position Type" key instead of an inline string(obviously error-prone)? 
Can't think of an obvious/slick way, but worth asking....